9.secretstore/deploy/components/statestore.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
We don't need this file as we are not using statestore component.

Reply:
fixed